backend/src/agents/__init__.py

```diff
@@ -10,26 +11,40 @@
 config = Config()
 
 
-validator_agent = ValidatorAgent(config.validator_agent_llm)
-intent_agent = IntentAgent(config.intent_agent_llm)
-answer_agent = AnswerAgent(config.answer_agent_llm)
+def get_validator_agent() -> Agent:
+    return ValidatorAgent(config.validator_agent_llm)
 
 
-def get_agent_details(agent):
+def get_intent_agent() -> Agent:
+    return IntentAgent(config.intent_agent_llm)
+
+
+def get_answer_agent() -> Agent:
+    return AnswerAgent(config.answer_agent_llm)
+
+
+def agent_details(agent) -> dict:
     return {"name": agent.name, "description": agent.description}
 
 
-agents = [DatastoreAgent(config.datastore_agent_llm), MathsAgent(config.maths_agent_llm)]
-agents_details = [get_agent_details(agent) for agent in agents]
+def get_question_agents() -> List[Agent]:

```

Comment:
I wonder if the name question_agents is too vague, could it be something like available_agents?

Reply:
yeah I like available_agents, will change to that